contributions/essay/lhagm/README.md

```diff
@@ -0,0 +1,21 @@
+# Assignment Proposal
+
+## Title
+
+Devops vs agile
+
+## Names and KTH ID
+
+- Linnea Hagman (lhagm@kth.se)
+
+## Deadline
+
+Task 4
+
+## Category
+
+Essay
+
+## Description
+
+I will erite an essay about agile vs devops. What are the similarities and differences? Can they complement eachother? Should you prefer one over the other?
```

Comment:
Elaborate more on the complementarity of Agile and DevOps practices. Include a Headline of the topics that you plan to cover in the essay.